src/auth/user/user.controller.ts

```diff
@@ -68,6 +115,22 @@ export class UserController {
 
   // Method used in tests to set the username in the session
   @Get('user/test/:username')
+  @ApiBasicAuth('Tests')
+  @ApiOperation({
+    summary: 'Set username for tests',
+    description: 'Used in tests to set the username in the session',
+  })
+  @ApiHeader({
+    name: 'authorization',
+    description: 'The base64 encoded GitHub username and password',
```

Comment:
This also requires the `Basic` authorization prefix?

Reply:
done